content/docs/user-guide/running-dvc-on-windows.md

```diff
@@ -5,22 +5,31 @@ involving system performance. Some, for example, have to do with NTFS file
 system characteristics and Windows built-in security mechanisms. Below are some
 workarounds that can help avoid these potential problems:
 
-## POSIX-like command line shell
-
-Many of the DVC commands have POSIX-style options that are given with a double
-dash `--`. This isn't supported by the simple Windows command prompt `cmd`.
-
-Common Windows terminal alternatives are
-[Git Bash](https://gitforwindows.org/#bash) or
-[Anaconda Prompt](https://docs.anaconda.com/anaconda/user-guide/getting-started/#open-prompt-win)
-– but they may not support all the POSIX features (e.g. `\` line continuation).
-
-💡 We recommend the full [Cmder](https://cmder.net/) console emulator (which
-already includes _Git for Windows_).
-
-Its also possible to enjoy a full Linux terminal experience with the
-[WSL](https://blogs.windows.com/windowsdeveloper/2016/03/30/run-bash-on-ubuntu-on-windows/)
-– but it may not be possible to access GPUs from this subsystem.
+## POSIX-like command line experience
+
+The regular Command Prompt (`cmd`) in Windows will most likely not help you use
+DVC effectively, or follow the examples in our docs. Please avoid it. There's no
+perfect solution, bu here are some ideas:
+
+- The full [Cmder](https://cmder.net/) console emulator combines several useful
+  tools like [ConEmu](https://conemu.github.io/), and
+  [Git for Windows](https://gitforwindows.org/)\* (Git Bash) among other
+  [shell options](https://github.com/cmderdev/cmder/blob/master/README.md#access-to-multiple-shells-in-one-window-using-tabs).
+- [Anaconda Prompt](https://docs.anaconda.com/anaconda/user-guide/getting-started/#open-prompt-win)
+  is another recommendation, but it may not support all the desired CLI features
+  (e.g. `\` line continuation).
+- Consider enabling and using
+  [WSL](https://blogs.windows.com/windowsdeveloper/2016/03/30/run-bash-on-ubuntu-on-windows/)
+  ([Windows Terminal](https://devblogs.microsoft.com/commandline/) also
+  recommended). But it may be slow and unable to access GPUs, et al.\*
+- Install an actual Linux distro (e.g. Ubuntu) on a virtual machine, or in a HD
+  partition (dual boot).
+
+> \* Please consult this [solution](https://stackoverflow.com/a/1257613/761963)
```

Comment:
def would not put it here - this is not DVC related in any way

Reply:
I know but it's an issue I struggled with from the go with Cmder just trying to use `git status`... Can't even commit changes because of this.

Reply:
make it single link close to Cmder ... but I would avoid this at all ... we can't cover all edge case - that's what SO for

Reply:
Yeah I just removed it but pretty sure it's something most people will battle with. Oh well, not our fault that Windows sucks